definitions/features/redis.rb

```diff
@@ -0,0 +1,22 @@
+class Features::Redis < ForemanMaintain::Feature
+  SCL_PREFIX = 'rh-redis5'.freeze
+  SERVICE_NAME = "#{SCL_PREFIX}-redis".freeze
```

Comment:
This could be fine for now, but will be an issue for Debian if ever used there. This will also break on EL8 when we land there. Unfortunately other SCLs have `syspaths` packages to allow using the classic service names. Redis does not appear to. @evgeni have you seen anything? would it be worth our shipping our own?

Reply:
We can ask the SCL team to provide one? :)